docs/extensions/logging-and-monitoring.md

```diff
@@ -128,7 +138,12 @@ data:
         Reserve_Data        True
 ```
 
-:information: It's a good idea to put the logging configuration into the Helm chart that also deploys the extension controller while the monitoring configuration can be part of the Helm chart/deployment routine that deploys the provider-specific component into the shoot namespace.
+Further details how to define parsers and use them with examples can be found in the following [guide](https://github.com/gardener/gardener/blob/master/docs/development/log_parsers.md).
+
+## Tips
+
+- Container names inside pods must be prefixed with the extension name. Fluent-bit parses the logs based on the container name and this will make them distinguishable.
```

Comment:
Could you elaborate on this tip? Probably add some examples.
I think get the point, but I am not sure. For example, if a seed is hosting GCP and AWS shoots, both provider extensions will configure fluent bit to parse `cloud-controller-manager` logs and in case the GCP and AWS CCMs are using different logging format, fluent bit needs different configurations for these CCMs, right?

Reply:
Please, check again this one.

Reply:
Actually the filter takes `pod_name`*`container_name`

In the case with the `cloud-controller-manager`:
If both pods have the same log format we can use the same filter for them.
If they have different log format:
  1) Firstly they should have different `container_name` or `pod_name` because they must differ by the `fluent-bit`
  2) Then we should create two different filters for them and to match them